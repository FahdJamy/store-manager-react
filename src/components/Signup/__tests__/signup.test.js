import React from 'react';
import { shallow } from 'enzyme'

import SignupForm from '../Signup';

describe('tests SignupForm', ()=> {
    it ('it should render without rashing', () => {
        let wrapper = shallow(<SignupForm />)
        expect(wrapper).toMatchSnapshot()
    });
});
