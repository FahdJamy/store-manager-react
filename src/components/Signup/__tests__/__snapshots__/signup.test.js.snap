// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests SignupForm it should render without rashing 1`] = `ShallowWrapper {}`;
